build.gradle.kts

```diff
@@ -147,6 +147,21 @@ publishing {
 						name.set("Sebastiaan de Schaetzen")
 						email.set("sebastiaan.de.schaetzen@gmail.com")
 					}
+					developer{
+						id.set("TheBusyBiscuit")
+						name.set("TheBusyBiscuit")
+						email.set("mrCookieSlime@gmail.com")
```

Comment:
Please don't just put my email there...
If anything use the public one that's linked everywhere... `slimefuncontact@gmail.com`.
But I much prefer not having any email set for me.